packages/flutter/lib/fix_data.yaml

```diff
@@ -17,6 +17,65 @@
 
 version: 1
 transforms:
+  - title: 'Remove BoxShape.rectangle'
```

Comment:
Before each title can you add a comment linking to this PR? Other fixes should have the same notation for reference.